### PR TITLE
Mmg 5.2.5 (new formula)

### PR DESCRIPTION
**Pull request by Algiane**
The Mmg platform gathers software dedicated to simplicial remeshing.

### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ] Removed the `revision` line, if any, when bumping the version number?
- [ ] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ ] Not altered the `bottle` section?
- [ ] Checked that the tests still pass?

### CODE REVIEW

File: mmg.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class Mmg < Formula
+  desc "Simplicial remeshers: mesh improvement,"\
+  "isotropic or anisotropic mesh adaptation and isovalue"\
+  "discretization."
```
Comment (jonchang) [inline mmg.rb]: Please shorten this description. `title` + `desc` should be less than 80 characters and this is 113.

File: mmg.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  homepage "https://www.mmgtools.org"
+  url "https://github.com/MmgTools/mmg/archive/v5.2.5.tar.gz"
+  sha256 "a39f82fe41dcc59fda92c58634397da527de6ca3ee3b99127b598a8ffebfd301"
```
Comment (jonchang) [inline mmg.rb]: Please add a `head` url pointing to the github repository. Please add `# tag` and `# doi` annotations if applicable (see `raxml.rb` for an example).

File: mmg.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      end
+
+      system "make"
+      system "make", "install"
+      system "ctest", "-VV" if build.with? "test"
+    end
+  end
+
+  test do
+    system "ctest"
```
Comment (jonchang) [inline mmg.rb]: This test fails:

```
==> ctest
Failed to execute: ctest
Error: homebrew/science/mmg: failed
Failed executing: ctest 
```

File: mmg.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
+class Mmg < Formula
+  desc "Simplicial remeshers: mesh improvement,"\
+  "isotropic or anisotropic mesh adaptation and isovalue"\
+  "discretization."
+  homepage "https://www.mmgtools.org"
+  url "https://github.com/MmgTools/mmg/archive/v5.2.5.tar.gz"
+  sha256 "a39f82fe41dcc59fda92c58634397da527de6ca3ee3b99127b598a8ffebfd301"
+
+  option "with-testing", "Run the quick tests"
```
Comment (jonchang) [inline mmg.rb]: If they are indeed quick then these should probably be run by default. Try this wording:

```ruby
option "without-test", "Disable build-time checking (not recommended)"
```

**Comment by Algiane** [comment]
Hi @jonchang,
Thank you for the feedback. It shoud be better now.
Please, tell me if there is something else to fix.
Regards,
Algiane

**Comment by iMichka** [comment]
@BrewTestBot test this please

**Comment by iMichka** [comment]
@BrewTestBot test this please

**Comment by iMichka** [comment]
We updated our "new formula" policy for homebrew-science:

> We will no longer accept new formula in homebrew-science, as this tap is being phased out. Please consider submitting your formulae to https://github.com/Homebrew/homebrew-core or host it in your own tap (https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap.html).

I am closing this pull request, if you need any advice please feel free to add a comment in this issue. Thanks!

**Pull request closed without merging**